### PR TITLE
use the setuptools.setup package_data parameter

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

**Comment by 0x00b1** [comment]
@mcquin ready for review

File: tests/modules/__init__.py [hunk 1]
```diff
@@ -25,7 +25,8 @@
 
 __temp_example_images_folder = None
 
-cp_logo_url = "https://raw.githubusercontent.com/CellProfiler/CellProfiler/master/artwork/CP_logo.png"
+# FIXME: use a stable URI
+cp_logo_url = "https://raw.githubusercontent.com/CellProfiler/CellProfiler/0c64193dd9108934400494fffa41d24f3df1573c/artwork/CP_logo.png"
```
Comment (mcquin) [inline tests/modules/__init__.py]: `http://cellprofiler.org/images/CPLOGO.png`

Comment (0x00b1) [inline tests/modules/__init__.py]: Unfortunately, it expects the shape `(70, 187, 3)`:

https://github.com/CellProfiler/CellProfiler/pull/2806/files#diff-8e024a4647cd7ab59604495f60825c9dL30

Comment (mcquin) [inline tests/modules/__init__.py]: 👍

**Comment by 0x00b1** [comment]
@mcquin fixed the setup.cfg and maifest.in issues! Thanks!

**Pull request merged**